packages/frontend/src/state/bull/utils.ts

```diff
@@ -70,3 +70,8 @@ const calcCrabToMint = (depositEthAmount: BigNumber, strategyCollat: BigNumber,
 
   return totalSupply.times(depositorShare).div(new BigNumber(1).minus(depositorShare))
 }
+
+// https://docs.euler.finance/developers/numeric-limits#interestrate
+export const getEulerInterestRate = (apy: BigNumber) => {
```

Comment:
@aleone

Reply:
Already mentioned in the previous comment. We are using the appropriate APY from Euler contracts